com.warlordsoftwares.jpeg2pdf.yml

```diff
@@ -0,0 +1,42 @@
+app-id: com.warlordsoftwares.jpeg2pdf
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-23.08'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+command: jpeg2pdf.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfsd
+  - --device=dri
+  - --socket=pulseaudio
+cleanup:
+  - /include
+  - /lib/pkgconfig
+modules:
+  - python3-pikepdf.json
+  - python3-req.json
+  - name: jpeg2pdf
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/jpeg2pdf
+      - cp -r * ${FLATPAK_DEST}/jpeg2pdf
+      - install -Dm755 ${FLATPAK_DEST}/jpeg2pdf/jpeg2pdf.sh ${FLATPAK_DEST}/bin/jpeg2pdf.sh
+      - install -Dm644 ${FLATPAK_DEST}/jpeg2pdf/share/icons/icon.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/com.warlordsoftwares.jpeg2pdf.png
+      - install -Dm644 ${FLATPAK_DEST}/jpeg2pdf/share/applications/jpeg2pdf.desktop ${FLATPAK_DEST}/share/applications/com.warlordsoftwares.jpeg2pdf.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/com.warlordsoftwares.jpeg2pdf.desktop
+      - install -Dm644 ${FLATPAK_DEST}/jpeg2pdf/share/metainfo/com.warlordsoftwares.jpeg2pdf.metainfo.xml ${FLATPAK_DEST}/share/metainfo/com.warlordsoftwares.jpeg2pdf.metainfo.xml
+    sources:
+      - type: archive
+        url: https://github.com/rishabh3354/FLATPAK_JPEG2PDF/archive/refs/tags/2024.2.1.tar.gz
```

Comment:
Since the appdata says project license is GPL, I'm assuming this won't be gone once the Flatpak is up on Flathub?

Reply:
Its GPL, open source code is available on github. It won't give any issue. People are using this software since 2019 as Snap package. Thanks